Makefile

```diff
@@ -1038,3 +1038,23 @@ dev-lab-kwok: ## Deploy kwok
 .PHONY: help
 help: ## Shows the available commands
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-40s\033[0m %s\n", $$1, $$2}'
+
+#####################
+# Bump new versions #
+#####################
+
+RELEASE_VERSION ?= 3.1.0-rc.7
```

Comment:
Suggest rename it to `KYVERNO_CHART_VERSION`, and add `POLICIES_CHART_VERSION` to differentiate the two charts.